client/Lykke.Service.Subscribers.Client/SubscribersClient.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Common.Log;
+using Lykke.Service.Subscribers.Client.AutorestClient;
+using Lykke.Service.Subscribers.Client.AutorestClient.Models;
+using Lykke.Service.Subscribers.Client.Models.ResponseModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.Subscribers.Client
+{
+    public class SubscribersClient : ISubscribersClient, IDisposable
+    {
+        private readonly ILog _log;
+        private SubscribersAPI _apiClient;
+
+        public SubscribersClient(string serviceUrl, ILog log, int timeout)
```

Comment:
Timeout type should be TimeSpan with default value. This way we create self-described contract.

Reply:
I've seen this in other services as a practice and I used to use you as an example! 
![image](https://user-images.githubusercontent.com/31039385/33515424-e3d176b6-d76b-11e7-8ed2-21cb03fc6c09.png)
What do you exactly want to change, why this is a bad practice?

Reply:
I get timeout value form settings file